dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.app.manager.wab.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = com.ibm.wsspi.appserver.app.manager.wab-1.0
+visibility = private
+-features=com.ibm.websphere.appserver.eeCompatible-8.0
```

Comment:
This needs to tolerate 6.0 and 7.0 for eeCompatible to be able to load with servlet-3.0 and servlet-3.1

Reply:
Thanks. I added tolerates eeCompatible 6.0,7.0 to the feature.